h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -1316,6 +1427,82 @@ private Chunk chk_xnew(Chunk[] chks, int c) {
     }
   }
 
+  // Perform de-standardization over matrix A for non NA values only
+  private static class DeStandardizeA extends MRTask<DeStandardizeA> {
```

Comment:
Seems like this class is exactly the same as `StandardizeA`, only uses different `normSub`. If you create `deSub` array similarly how you had `deMul` above, then it should be possible to remove this class altogether